modules/battery_manager/help/battery_manager.md

```diff
@@ -0,0 +1,54 @@
+# Battery Manager
+*For Rida: Add what each value in the test_battery does and what happens if it's left empty*
```

Comment:
?

Reply:
Yeah, same thing. I don't think I even bothered to look at this readme lol. @ridz1208, any ideas?

Reply:
@ridz1208 ??

Reply:
Someone (probably me) will have to look into what each field in the test_battery table does...

will get to it as soon as I  can

Reply:
@ridz1208 any updates?

Reply:
@ridz1208